src/app/lib/utilities/passport/index.js

```diff
@@ -0,0 +1,33 @@
+import pathOr from 'ramda/src/pathOr';
+
+export const getPassportHome = pageData => {
+  const homeServiceUrl = pathOr(
+    null,
+    ['metadata', 'passport', 'home'],
+    pageData,
+  );
+  return pageData && homeServiceUrl
```

Comment:
can just be

```js
return homeServiceUrl && homeServiceUrl
        .split('/')
        .pop()
        .toLowerCase()
```

because `homeServiceUrl` defaults to `null` if `pageData` is `undefined`. Remove the `.slice(-1)` too as it's not necessary to create a new array after the split method.